kitsune/users/static/users/js/actions/UserAuthActions.es6.js

```diff
@@ -1,149 +0,0 @@
-/* globals $:false */
```

Comment:
This file is referenced in kitsune/questions/static/questions/js/components/UserAuth.jsx
which is referenced in kitsune/questions/static/questions/js/components/AAQApp.jsx
which is referenced in kitsune/questions/static/questions/js/aaq.browserify.js
which doesn't seem to be referenced anywhere

There's a lot of AAQ react stuff to tear out... I wonder if it should be done as a separate PR.